app/views/base/_avatar.html.haml

```diff
@@ -1,6 +1,6 @@
 -id ||= nil
--kind ||= user.avatar_kind
 -size ||= "medium"
+-kind ||= user.avatar_kind
```

Comment:
Mix.. I really don't like the image so I deleted it.

It's ok to have kind. I know it is needed. but you don't need to specify it when asking the object for a valid image url.